backend/src/app.js

```diff
@@ -4,6 +4,8 @@ const cors = require('cors')
 
 const Usuario = require('./modelos/usuario');
 const repositorioUsuarios = require('./repositorios/repositorioUsuarios');
+const Solicitud = require('./modelos/solicitud');
+const repositorioSolicitud = require('./repositorios/repositorioDeSolicitudes');
```

Comment:
```suggestion
const repositorioDeSolicitudes = require('./repositorios/repositorioDeSolicitudes');
```

Lo cambiaria para que siga la convención de Usuarios